source/includes/gcp/_k8_pods.md

```diff
@@ -0,0 +1,918 @@
+            "volumeMounts": [
+              {
+                "mountPath": "/etc/kubernetes/scheduler.conf",
+                "name": "kubeconfig",
+                "readOnly": true
+              }
+            ]
+          }
+        ],
+        "dnsPolicy": "ClusterFirst",
+        "enableServiceLinks": true,
+        "hostNetwork": true,
+        "nodeName": "docker-desktop",
+        "priority": 2000000000,
+        "priorityClassName": "system-cluster-critical",
+        "restartPolicy": "Always",
+        "schedulerName": "default-scheduler",
+        "securityContext": {},
+        "terminationGracePeriodSeconds": 30,
+        "tolerations": [
+          {
+            "effect": "NoExecute",
+            "operator": "Exists"
+          }
+        ],
+        "volumes": [
+          {
+            "hostPath": {
+              "path": "/etc/kubernetes/scheduler.conf",
+              "type": "FileOrCreate"
+            },
+            "name": "kubeconfig"
+          }
+        ]
+      },
+      "status": {
+        "conditions": [
+          {
+            "lastTransitionTime": "2020-01-21T15:05:03.000-05:00",
+            "status": "True",
+            "type": "Initialized"
+          },
+          {
+            "lastTransitionTime": "2020-01-21T15:05:06.000-05:00",
+            "status": "True",
+            "type": "Ready"
+          },
+          {
+            "lastTransitionTime": "2020-01-21T15:05:06.000-05:00",
+            "status": "True",
+            "type": "ContainersReady"
+          },
+          {
+            "lastTransitionTime": "2020-01-21T15:05:03.000-05:00",
+            "status": "True",
+            "type": "PodScheduled"
+          }
+        ],
+        "containerStatuses": [
+          {
+            "containerID": "docker://05c676485218c2d6ff961349574ce209eeac001bf7d5468698619f8ce372751b",
+            "image": "k8s.gcr.io/kube-scheduler:v1.14.8",
+            "imageID": "docker-pullable://k8s.gcr.io/kube-scheduler@sha256:54ba7d0b618ca22d2a86ae31eb810ebda110045fe4eeb15d8777a4252a01ccb3",
+            "lastState": {
+              "terminated": {
+                "containerID": "docker://219bbc98442ab8ab426abbf4f85d38c30873e71afc417eb1a2670d308c93b25b",
+                "exitCode": 255,
+                "finishedAt": "2020-01-21T15:04:35.000-05:00",
+                "reason": "Error",
+                "startedAt": "2020-01-20T10:28:06.000-05:00"
+              }
+            },
+            "name": "kube-scheduler",
+            "ready": true,
+            "restartCount": 7,
+            "state": {
+              "running": {
+                "startedAt": "2020-01-21T15:05:05.000-05:00"
+              }
+            }
+          }
+        ],
+        "hostIP": "192.168.65.3",
+        "phase": "Running",
+        "podIP": "192.168.65.3",
+        "qosClass": "Burstable",
+        "startTime": "2020-01-21T15:05:03.000-05:00"
+      }
+    }
+  ],
+  "metadata": {
+    "recordCount": 3
+  }
+}
+```
```

Comment:
I feel the examples are just too big. It will look weird. can we reduce it?

Reply:
These are simplified real example. We can talk on how we can make them shorter.

Reply:
fixed